tools/klee/main.cpp

```diff
@@ -1110,7 +1100,8 @@ linkWithUclibc(StringRef libDir, std::string opt_suffix,
     replaceOrRenameFunction(modules[i].get(), "__libc_fcntl", "fcntl");
   }
 
-  createLibCWrapper(modules, EntryPoint, "__uClibc_main");
+  if (mainFn)
+    createLibCWrapper(modules, "main", "__uClibc_main");
```

Comment:
It will always crash on assert(userMainFn && "unable to get user main"). If use `--libc=uclibc --posix-runtime` and choose not `main` entry point because now `preparePOSIX` always rename `main` to `__klee_posix_wrapped_main`, but it still has pointer to it in `mainFn`.